static/app/views/issueDetails/streamline/foldSection.tsx

```diff
@@ -156,17 +170,16 @@ export function FoldSection({
       trackAnalytics('issue_details.section_fold', {
         sectionKey,
         organization,
-        open: !isCollapsed,
+        open: expanded,
         org_streamline_only: organization.streamlineOnly ?? undefined,
       });
-      setIsCollapsed(!isCollapsed);
+      setIsCollapsed(expanded);
```

Comment:
This broke my brain for a moment, but I see that the logic checks out. Might be nice to have a comment here.

Reply:
I'm going to delete all of it, just doing a very progressive change :D